src/researchhub_comment/tests/test_comments.py

```diff
@@ -368,29 +386,37 @@ def test_censor_post_comments_updates_discussion_count(self):
         )
         post_id = post_res.data["id"]
 
-        # Create a parent comment with multiple children
+        # Create a parent comment with a child
         parent_comment = self._create_post_comment(post_id, self.user_1)
-        child1 = self._create_post_comment(
+        _ = self._create_post_comment(  # noqa: F841
```

Comment:
No need to make a `_` assignment and mark it with `noqa`. Just don't assign and it will be fine.

```suggestion
        self._create_post_comment(
```